README.md

```diff
@@ -133,6 +135,10 @@ func main() {
 	// Print full date like Thursday 22 Aug 2016
 	fmt.Println(randomdata.FullDate())
 
+	// Get a full profile and use the data in it
```

Comment:
Would probably want this to say
"Get a complete and randomised profile of data generally used for users"